azure/azure-build-pipeline.yml

```diff
@@ -1,16 +1,7 @@
 name: "$(SourceBranchName)+$(BuildID)"
 
-trigger:
-  branches:
-    include:
-      - tags/refs/v*
-  tags:
-    include:
-      - v*
-
-pr:
-  branches:
-    include: ['*']
+trigger: none
```

Comment:
with this the pipeline will not trigger unless triggered manually - we want to keep this so that a version bump + tag will deploy the spec

Reply:
Restored this back to default for this pipeline.